content/articles/2023/2023-01-15_rust_kernel_module_part2.md

```diff
@@ -3,26 +3,26 @@ Tags: rust, kernel, Linux, ArchLinux
 Language: en
 Summary: Trying to build a hello world out-of-tree Rust kernel module for Linux 6.1. Part two.
 
-In my last [blog posts](/building-an-out-of-tree-rust-kernel-module.html) I
+In my last [blog posts](/building-an-out-of-tree-rust-kernel-module.html), I
```

Comment:
This one I'm not sure about. But I saw another typo:
```suggestion
In my last [blog post](/building-an-out-of-tree-rust-kernel-module.html), I
```

Reply:
> This one I'm not sure about.

AFAIK, in English one _usually_ sets a comma, when a part of the sentence has been "moved" to the front instead of staying in it's usual location according to the "SVOPT" order. (**s**ubject, **v**erb, **o**bject, [**p**lace], [**t**ime]) "In my last blog post" seems like a time or place indication.

> But I saw another typo

:+1: I didn't spot that one.